Assets/UI/DiplomacyDealView_CQUI.lua

```diff
@@ -1,3 +1,4 @@
+print("*** CQUI: DiplomacyDealView_CQUI loaded");
```

Comment:
do you want to keep this debug print?

Reply:
yeah, for this and the CityBannerManager, because they no longer use the "ReplaceUIScript" stuff in the modinfo file.  The debugging of these files in this new setup from Firaxis is less reliable, at least in terms of having error messages actually show up int he Live Tuner, so this is an ok workaround as it should only print that message once.